frontend/src/components/OdhAppCard.tsx

```diff
@@ -84,31 +86,18 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
   };
 
   const dropdownItems = [
-    <DropdownItem
-      key="docs"
-      className="odh-dashboard__external-link"
-      href={odhApp.spec.docsLink}
-      target="_blank"
-      rel="noopener noreferrer"
-    >
-      View documentation
-      <ExternalLinkAltIcon />
+    <DropdownItem key="docs" href={odhApp.spec.docsLink} target="_blank" rel="noopener noreferrer">
+      View documentation <ExternalLinkAltIcon />
     </DropdownItem>,
   ];
 
   const badgeClasses = classNames('odh-card__partner-badge', {
     'm-warning': odhApp.spec.category === 'Third party support',
   });
 
-  const quickStartClasses = classNames('odh-dashboard__external-link', {
-    'm-hidden': !odhApp.spec.quickStart,
-    'm-disabled':
-      getLaunchStatus(odhApp.spec.quickStart || '', qsContext) === LaunchStatusEnum.Close,
-  });
-
   if (odhApp.spec.quickStart) {
     dropdownItems.push(
-      <DropdownItem key="quick-start" className={quickStartClasses} href="#" onClick={onQuickStart}>
+      <DropdownItem key="quick-start" href="#" onClick={onQuickStart}>
```

Comment:
😕 Why are we keeping `href="#"`? Just for the cursor styles?

Reply:
hmm, interesting, didn't notice this, let me check

Reply:
We can remove it, makes no difference.